Sources/SwiftInspectorVisitors/FileVisitor.swift

```diff
@@ -108,4 +127,10 @@ public struct FileInfo: Codable, Equatable {
   mutating func appendClasses(_ classes: [ClassInfo]) {
     self.classes.append(contentsOf: classes)
   }
+  mutating func appendEnums(_ enums: [EnumInfo]) {
+    self.enums.append(contentsOf: enums)
+  }
+  mutating func appendExtension(_ extensionInfo: ExtensionInfo) {
```

Comment:
Asking to learn: why does this method accept one extension when other methods accept multiple classes and structs?

Reply:
Because we can only parse one extension at a time since extensions are top-level only.

To illustrate, the following code compiles:
```
struct Foo {
  struct Bar {}
}
```

And the following code doesn't:
```
extension Foo {
  extension Bar {}
}
```